### PR TITLE
subquery/subquid tutorial

### PR DESCRIPTION
**Pull request by thea-exe**
I'll probably add 1-2 more examples but I went into depth with the first one, would like to see if it looks satisfactory first! I wrote this with first-time devs in mind.

### CODE REVIEW

**Comment by yangwao** [comment]
@vikiival 👀

**Comment by thea-exe** [comment]
@vikiival oh wait I think I was just showing that to explain stuff 😅

**Comment by thea-exe** [comment]
It doesn't actually query anything, I wrote what it does :P

File: docs/tutorials/querytutorial.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+## How to write a Query 
+
+KodaDot currently uses two types of indexers, [Subquery] and [SubSquid]. You can use this tutorial as a way to write queries for both indexers, however there are minimal differences that you should keep track of!
```
Comment (vikiival) [inline docs/tutorials/querytutorial.md:3]: missing links for subquery and subsquid

File: docs/tutorials/querytutorial.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+##### Keep in mind that there are some query variable differences (i.e 'first' in Subquery and 'limit' in SubSquid) between Subquery and Subsquid. When writing a query, make sure you are using variables intended for your indexer. For learning purposes, we will use the SubSquid playground.
+
+We'll go through the components of a query: the query itself, the parameters, and query variables.
+
+The query itself is what you're ultimately fetching, at the end of intializing your query variables, your query should be able to fetch and return the information you specifically asked for. 
+
+The parameters are what helps the query find what you need. If one of your parameters is 'id', then the query will look for id's.
```
Comment (vikiival) [inline docs/tutorials/querytutorial.md:11]:
```suggestion
The parameters are what helps the query find what you need. If one of your parameters is `id`, then the query will look for id's.
```

File: docs/tutorials/querytutorial.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+The query variables are what you initialize (for example, the address of your own account) and where the query goes to, to begin fetching information.
+
+Let's go through examples to show you how this works. If you'd like to understand how [general querying]() works, we have an article for that as well. In the meantime, these will be queries specific to Subquery/SubSquid.
```
Comment (vikiival) [inline docs/tutorials/querytutorial.md:15]: possibly empty link?

**Review by vikiival — CHANGES_REQUESTED** [review]
minor

**Pull request merged**